src/commontransformations.jl

```diff
@@ -0,0 +1,352 @@
+# Some common transformations are defined here
+
+###################
+### Translation ###
+###################
+
+immutable Translation{N, T} <: AbstractTransformation{FixedVector{N}, FixedVector{N}}
+    dx::Vec{N, T}
+end
+Translation(x,y) = Translation(Vec(x,y))
+Translation(x,y,z) = Translation(Vec(x,y,z))
+Base.show(io::IO, trans::Translation) = print(io, "Translation$(trans.dx._)")
+
+function transform{N}(trans::Translation{N}, x::FixedVector{N})
+    (x_promoted, dx_promoted) = promote(x, trans.dx) # Force same data type
+    x_promoted + (typeof(x_promoted))(dx_promoted) # Force same base type
```

Comment:
Seems a bit ick (due, I imagine to the general lack of some facilities in FSAs package).  How about adding a TODO or FIXME with a note about making this nicer once `+` works better with FSAs?